### DISCUSSION TITLE
Do we have a way to access prefetch data when hovering over links?

### DISCUSSION THREAD

**Discussion by meotimdihia** [vercel/next.js#30011] [Help] [answered]
Do we have a way to access prefetch data when hovering over links?
for example: when I hover to a link next.js prefetch to `/_next/data/D1vs16ncRHFStJyZUL4gH/game/1.json`
I want to get the data from the above link to show a popup or preview.

**Comment by jamesmosier** [comment, marked as answer]
There is currently no way to get the prefetched data as you described.
